concepticondata/conceptlists.tsv

```diff
@@ -193,3 +193,4 @@ Stokhof-1975-117	Stokhof	1975		117	basic,areal	English	New Guinea	A "representat
 Clouse-1993-71	Clouse	1993		71	basic,areal	English	New Guinea	A wordlist designed to elucidate the relationships between Kirikiri and other Lakes Plains languages	Clouse1993	Clouse1993		29-30	
 Smith-2015-97	Smith	2015		91	basic,areal	English	Borneo	A 97 item wordlist designed to demonstrate the differences and relationships between the Kenyah and Kayan languages of Borneo	Smith2015	Smith2015		351-355	
 Laycock-1968-50	Laycock	1968		50	basic,areal	English	New Guinea	A 50 item wordlist	Laycock1968	Laycock1968		51-363	
+Larson_and_Larson-1972-68	Larson & Larson	1972		68	basic,areal	English	New Guinea	A 68 item wordlist based on Swadesh 100	Larson_and_Larson1972	Larson_and_Larson1972		90-92	
```

Comment:
sorry for bothering, but I'd restate this as Larson-1972-68, according to our conventions (we only name first author). And in the note field, you should reference like this:
```text
A 68 item wordlist based on [Swadesh 100](:ref:Swadesh-1955-100).
```
In this way, there'll be a hyperlink in the web-app.